src/clients/identity/token-resolvers/access-token-services/cache-services/cache-user-access-token.service.ts

```diff
@@ -0,0 +1,18 @@
+import { ICacheManager } from '../../../../../cache/cache.manager.interface';
+import { IEmptyAccessToken, IEntityWithRoles, IUserAccessToken, tokenTypes } from "../../../types";
+import { AccessTokenService } from '../services/access-token.service';
+import { CacheAccessTokenService } from './cache-access-token.service';
+
+export class CacheUserAccessTokenService extends CacheAccessTokenService<IUserAccessToken> {
+    constructor(
+        public readonly entityCacheManager: ICacheManager<IEntityWithRoles | IEmptyAccessToken>,
+        public readonly activeAccessTokensCacheManager: ICacheManager<string[]>,
+        public readonly userAccessTokenService: AccessTokenService<IUserAccessToken>,
+    ) {
+        super(entityCacheManager, activeAccessTokensCacheManager, userAccessTokenService, tokenTypes.UserAccessToken);
+    }
+
+    protected getCachePrefix(): string {
+        return 'tenant_access_tokens_'
```

Comment:
i suggest to have also Frontegg-SDK prefix as well. specially for redis as used by others as well

Reply:
done